### PR TITLE
STORY-17249 Findlaw request to add obfuscated_tag_id and destination_phone_number to the GET response for the Network Integration Ringpool API

### PR DESCRIPTION
**Pull request by abhig10**
[STORY-17249 Findlaw request to add obfuscated_tag_id and destination_phone_number to the GET response for the Network Integration Ringpool API](https://invoca.atlassian.net/browse/STORY-17249)

Index page.

Screenshot 2024-01-29 at 6 59 33 PM

Get Ring Pools response update.

Screenshot 2024-01-29 at 6 57 33 PM

Get Ring Pool response update.

Screenshot 2024-01-29 at 6 57 42 PM

Post Ring Pool request update.

Screenshot 2024-01-29 at 6 57 50 PM

## Checklist

- [ ] Find the [Service owning team](https://docs.google.com/spreadsheets/d/1YF2wuepY5SZTpVEdT9gwhTPhpnDGZeVjywEwvbJS5TQ/edit#gid=0&fvid=1520238175) for these changes, and tag the team as "Reviewers" on this PR
- [ ] Test the documentation changes on readthedocs as a private branch
- [ ] If changing general content, have agreement on whether to apply to latest version or all versions (if all versions, provide links to the related PRs below)

### CODE REVIEW

File: source/api_documentation/network_integration/ringpools/index.rst [hunk 1]
```diff
@@ -67,6 +67,18 @@ By default, RingPools will capture params based on your Marketing Data Dictionar
 
       If not passed when creating a new RingPool, the RingPool will be created using the default setting of true.
 
+  * - customer_phone_numbers
+    - Array of hashes
+    - Only applicable for `GET /ring_pools/<ring_pool_id>` API endpoint.
```
Comment (BMTTMC) [inline source/api_documentation/network_integration/ringpools/index.rst:72]: ❓  These hashes are only present in GET responses.  But the `destination_phone_number` is already in use in POST requests.  It's even mentioned in the documentation (line [123](https://github.com/Invoca/developer-docs/pull/372/files#diff-06ceffdc1dda79ba4fbeb0033fdb8077b98fcdf8d54941cd8ea6126994fa87e7R123)).  Should it be added for POST?

**Review by BMTTMC — COMMENTED** [review]
Changes look good, just one question.

Once this is done, you'll also need to create PRs for the other supported versions.

File: source/api_documentation/network_integration/ringpools/index.rst (hunk 1 above)
Comment (abhig10) [reply source/api_documentation/network_integration/ringpools/index.rst:72]: > Should it be added for POST?

Do you mean in the separate table below [POST heading](https://developers.invoca.net/en/latest/api_documentation/network_integration/ringpools/index.html#post)? I can do that.

Comment (abhig10) [reply source/api_documentation/network_integration/ringpools/index.rst:72]: > Once this is done, you'll also need to create PRs for the other supported versions.

[STORY-16732](https://invoca.atlassian.net/browse/STORY-16732) only added the `obfuscated_tag_id` documentation for the latest API version, so this maybe a question for the team, if we want to include previous versions (`2019-05-01` and above) to have this documentation or update the API to only work for `2022-08-01` version and above? I'm going to put a message in team channel to go over this.

Slack 🧵 https://invoca.slack.com/archives/C05N0QGR66Q/p1706543796487589

Comment (abhig10) [reply source/api_documentation/network_integration/ringpools/index.rst:72]: Table under POST heading showing the parameter documentation.

Screenshot 2024-01-29 at 9 43 52 PM

**Review by BMTTMC — APPROVED** [review]
Looks good to me!  I'll keep the discussion around other versions outside this one. 👍

**Comment by abhig10** [comment]
@omeedrj Please merge and deploy this when you get a chance.

**Pull request merged**